index.html

```diff
@@ -2237,6 +2243,13 @@ <h2 id="AudioContext">
             </dt>
             <dd>
               <p>
+                If the <a>relevant settings object</a>'s <a>responsible
```

Comment:
Don't quite follow "relevant settings object" and "responsible document".

Plus these `<a>` tags don't seem to link up anything, at least in preview.

Reply:
Relevant settings objects are always "of" something. Since the object is not created yet here, the concept isn't really applicable. You want "current settings object" here. See https://html.spec.whatwg.org/#realms-settings-objects-global-objects in particular

> Note that in constructors, where there is no this value yet, the current concept is the appropriate default.